tests/test_prep_6_10_upgrade.py

```diff
@@ -0,0 +1,31 @@
+from testfm.log import logger
+
+
+def test_positive_prep_6_10_upgrade(ansible_module):
+    """Runs content directory preparation for pulp3 migration.
+
+    :id: 46531c4a-2276-4ccd-aed7-92c53d1ad267
+
+    :setup:
+        1. foreman-maintain should be installed.
+
+    :steps:
+        1. Create /var/lib/pulp/content directory with root owner and 0000 mode
+        2. Run foreman-maintain prep-6.10-upgrade
+
+    :expectedresults: Owner group and mode should be changed.
+
+    :CaseImportance: High
+    """
+    content_dir = "/var/lib/pulp/content"
+    ansible_module.file(path=content_dir, state="directory", owner="root", mode="0000")
+
+    contacted = ansible_module.command("foreman-maintain prep-6.10-upgrade")
+    for result in contacted.values():
+        logger.info(result["stdout"])
+        assert "FAIL" not in result["stdout"]
+        assert result["rc"] == 0
+
+    contacted = ansible_module.stat(path=content_dir)
```

Comment:
```suggestion
    contacted = ansible_module.stat(path="/var/lib/pulp/content")
```

Reply:
I don't know, I would rather keep it defined on one place. Why do you prefer passing it directly?

Reply:
yep, it is just to optimize the tests, if the line count goes > 100 then defining variables makes sense, otherwise, why add an extra step for variable assignment.
Again it's a suggestion, not insisting you change if you like you can skip :)